### PR TITLE
Add capability for sending emails via SMTP

### PR DESCRIPTION
**Pull request by acrellin**
Tested locally with my own gmail account

### CODE REVIEW

**Review by kmshin1397 — COMMENTED** [review]
Google blocked my machine from logging into my gmail when I tried to test it with my gmail as the `from_email`, citing something along the lines of "Less secure app access":

Screenshot 2020-11-16 185935

I'm guessing you didn't see this @acrellin ?

File: skyportal/email_utils.py
Comment (acrellin) [reply skyportal/email_utils.py]: Done

**Comment by acrellin** [comment]
@kmshin1397 yes, I had to "Allow less secure apps" on my Google account before the authentication would work. I'm still looking into how we can get SkyPortal to be seen as a secure app to Google

**Comment by acrellin** [comment]
It seems that any apps that login directly w/ username/password are considered "less secure" as they do not have 2FA capabilities.

**Comment by kmshin1397** [comment]
Hmm okay, I'm thinking that means we either have to use a different SMTP server, or there's probably a way to do this with the Gmail API? I think ideally, we don't want to settle for just disabling the security check on the gmail account.

**Comment by acrellin** [comment]
Yeah, that doesn't really have anything to do with this PR, though, as any
SMTP service can be used here

**Review by kmshin1397 — APPROVED** [review]

**Comment by kmshin1397** [comment]
That's true. I'll approve this. Do you want a second review on this, or should I go ahead and merge?

**Comment by acrellin** [comment]
@kmshin1397 I think this is small enough that you can go ahead and merge. Thanks!

**Pull request merged**
